app/enterprise/changelog.md

```diff
@@ -750,6 +750,14 @@ open-source **Kong Gateway 2.2.0.0**:
 - The `shorthands` attribute in schema definitions is deprecated in favor of
   the new `shorthand_fields` top-level attribute.
 
+## 2.1.4.5
+**Release Date** 2021/03/31
+
+### Fixes
+
+#### Enterprise
+Fixed an issue when upgrading Kong Enterprise Gateway from v1.5.x to v2.1.x. Before the fix, when admin consumers were shared across multiple workspaces, it was possible for the migration to fail. This happened because plugin entities that depend on consumer entities must live in the same workspace as the consumer entity. This fix migrates these plugin entities to the same workspace as the consumer entities. Customers affected by this issue will need to upgrade to at least v2.1.4.5 before attempting to migrate from v1.5.x to v2.1.x.
```

Comment:
> will need to upgrade to at least v2.1.4.5 before attempting to migrate from v1.5.x to v2.1.x.

This is confusing phrasing; "upgrade to 2.1.4.5 before upgrading to 2.1.x" doesn't really make sense. Do they need to upgrade 1.5.x to 2.1.4.5 and then to whatever other version? That is, we're telling them to skip all 2.1.x versions before 2.1.4.5.

Reply:
ok, I updated that line - let me know what you think